### ISSUE TITLE
mpld3_demo notebook fails on matplotlib 3.5.0

### ISSUE THREAD

**Issue by RussTedrake** [mpld3/mpld3#502]
matplotlib 3.5.0 is released (https://pypi.org/project/matplotlib/#history) and I believe `mpld3` needs an update.  The errors end with:

```
/usr/local/lib/python3.9/site-packages/mpld3/mplexporter/utils.py in get_axis_properties(axis)
    229         } 
    230         props['tickformat_formatter'] = "percent"
--> 231     elif isinstance(formatter, ticker.IndexFormatter):
    232         props['tickformat'] = [text.get_text() for text in axis.get_ticklabels()]
    233         props['tickformat_formatter'] = "index"

AttributeError: module 'matplotlib.ticker' has no attribute 'IndexFormatter'
```
You can reproduce the error by running the `mpld3_demo` notebook.

**Comment by titodalcanton** [comment]
@RussTedrake see https://github.com/mpld3/mplexporter/issues/62.

**Comment by vladh** [comment]
All should be fixed in 0.5.6. Could you please test and confirm that everything works? Thank you!

**Comment by RussTedrake** [comment]
Confirmed that everything works in my tests.  Many thanks!
